src/display/node_stream.js

```diff
@@ -359,17 +365,23 @@ class PDFNodeStreamRangeReader extends BaseRangeReader {
     }
     this._httpHeaders['Range'] = `bytes=${start}-${end - 1}`;
 
+    let handleResponse = (response) => {
+      if (response.statusCode === 404) {
+        const error = new MissingPDFException(`Missing PDF "${this._url}".`);
+        this._storedError = error;
```

Comment:
I think we should `return` after this too to avoid setting the readable stream to a failed response, just like above.

Reply:
Yes, that seems reasonable; fixed now.